txhelpers/txhelpers.go

```diff
@@ -979,27 +977,12 @@ const (
 )
 
 // AddressValidation performs several validation checks on the given address
-// string. Initially, decoding as a Decred address is attempted. If it fails to
-// decode, btcutil is used to try decoding it as a Bitcoin address. If both
-// decoding fails, AddressErrorDecodeFailed is returned with AddressTypeUnknown.
-// If the address is a Bitcoin address, AddressErrorBitcoin is returned with
-// AddressTypeBitcoin. If the address decoded successfully as a Decred address,
-// it is checked against the specified network. If it is the wrong network,
-// AddressErrorWrongNet is returned with AddressTypeUnknown. If the address is
-// the correct network, the address type is obtained. A final check is performed
-// to determine if the address is the zero pubkey hash address, in which case
-// AddressErrorZeroAddress is returned with the determined address type. If it
-// is another address, AddressErrorNoError is returned with the determined
-// address type.
```

Comment:
The other comments about `AddressErrorWrongNet`, etc. are still valid

Reply:
Right, but do we need to list all of the checks in the comment?  If you pass an invalid address, the error will tell you why.

Reply:
We are pulling a lot of code (btcsuite) to just tell the user they entered a Bitcoin address instead of a Decred address.   We don't care about bitcoin.

Reply:
Ok about bitcoin, but I think the comments describe the expected behavior and I don't want to remove them.